client/python/datajunction/client.py

```diff
@@ -583,33 +543,68 @@ def data(  # pylint: disable=too-many-arguments
                     "disabled. You can install pandas by running `pip install pandas`."
                 ),
             ) from exc
-        response = self._session.get(
-            "/data/",
-            params={
-                "metrics": metrics,
-                "dimensions": dimensions,
-                "filters": filters,
-                "engine_name": engine_name or self.engine_name,
-                "engine_version": engine_version or self.engine_version,
-            },
-        )
-        results = response.json()
-        if not response.ok:
-            raise DJClientException(f"Error retrieving data: {response.text}")
-        if results["state"] != "FINISHED":
-            raise DJClientException(
-                f"Query state {results['state']}, errors: {results['errors']}",
-            )
-        if not results["results"]:
-            raise DJClientException("No data returned for requested set")
-        columns = results["results"][0]["columns"]
-        rows = results["results"][0]["rows"]
-        return pd.DataFrame(rows, columns=[col["name"] for col in columns])
+
+        with alive_bar(
+            title="Processing",
+            length=20,
+            bar="smooth",
+            force_tty=True,
```

Comment:
I see this in their docs:
```
The values accepted are:

force_tty=True -> always enables animations, and auto-detects Jupyter Notebooks!
force_tty=False -> always disables animations, keeping only the final receipt
force_tty=None (default) -> auto detect, according to the terminal's tty state
```

Do you think True is safer enough for our most common uses? We can go with True and see how it goes.

Reply:
Yeah, I agree that `True` should be enough for most of our use cases. I don't think many folks will be using the DJ Python client from the Python interactive interpreter, whereas Jupyter is the most likely candidate. But we can always change it if that turns out to be wrong 😄